documentation/book/assembly-kafka-cluster.adoc

```diff
@@ -0,0 +1,39 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {ContextProduct}
+
+include::common/attributes.adoc[]
+
+[id='kafka-broker-{context}']
+= Kafka cluster
+
+When installing Kafka, {ProductName} also installs a Zookeeper cluster and adds the necessary configuration for the Kafka broker connection.
```

Comment:
It is not clear to me what Kafka broker connection do we mean here. Would this be better? _... adds the necessary configuration to connect Kafka with Zookeeper._